apidoc/Titanium/Database/DB.yml

```diff
@@ -73,7 +73,7 @@ properties:
       setting file properties.
     type: Titanium.Filesystem.File
     permission: read-only
-    platforms: [iphone,ipad]
+    platforms: [android,iphone,ipad]
     since: "2.0.0"
```

Comment:
Never had that before, but the docs seem to offer a way to set the `since` property platform-dependent:
```
    since:
        android: "2.0.0"
        iphone: "6.0.0"
        ipad: "6.0.0"
```